.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,32 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
```

Comment:
```suggestion
<!-- Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change. -->
```

Perhaps we can comment the descriptions so that if we forget to remove them, we don't show too much redundant information in the PR